packages/@uppy/provider-views/src/Breadcrumbs.js

```diff
@@ -1,9 +1,8 @@
-const { h } = require('preact')
+const { h, Fragment } = require('preact')
 
-// TODO use Fragment when upgrading to preact X
 const Breadcrumb = (props) => {
   return (
-    <span>
+    <Fragment>
```

Comment:
I believe this can be simplified even further by just using `<>` and `</>` instead of having to import Fragment